### ISSUE TITLE
about avx

### ISSUE THREAD

**Issue by ghost** [Barebit/x86reference#11]
Is avx instruction updated in this manual?

**Comment by BarebitOpenSource** [comment]
I don't have time to add them. We also need to design first how AVX instructions should be added because they use backward incompatible instruction format.

**Comment by ghost** [comment]
I am learning to see if I can contribute code to you.

**Comment by BarebitOpenSource** [comment]
Do you have experience with coding with AVX instructions?

The most important is figuring out AVX [instruction operand codes](http://ref.x86asm.net/#Instruction-Operand-Codes). These can be seen in the geek editions, see for example [`ADDPS Vps, Wps`](http://ref.x86asm.net/geek.html#x0F58). There are no AVX codes defined in the reference at the moment.

I think that Intel manual defines the AVX operand codes; also [sandpile.org](https://sandpile.org/) should help.

In short, we should start with a table of AVX operand codes, completing [this one](http://ref.x86asm.net/#Instruction-Operand-Codes), inspired by Intel manual and sandpile.org.

**Comment by ghost** [comment]
I don't have experience in avx instruction, but I just used the avx command. I refer to the official manual of intel.

**Comment by BarebitOpenSource** [comment]
I work on the VEX-encoded instructions now but not sure when it will be finished.

**Comment by BarebitOpenSource** [comment]
https://github.com/mazegen/x86reference/issues/8
